tfe/data_source_workspace.go

```diff
@@ -86,6 +86,11 @@ func dataSourceTFEWorkspace() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"remote_execution": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
```

Comment:
I tried to keep these parameters somewhat (not fully) alphabetically ordered. So in the spirit, this new parameter should probably be placed between `queue_all_runs` and `ssh_key_id`.